core-services/quay-mirroring/knative/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- smarterclayton
```

Comment:
If you have him at the top-level under `quay-mirroring` you can omit him in specific dirs

Reply:
These `OWNERS` are just copies of the existing ones from `cluster/ci/config/mirroring/`, haven't touched them. But yeah, it probably makes sense to remove these redundancies.

Reply:
Fixed.